flink-runtime/src/test/java/org/apache/flink/runtime/state/heap/CopyOnWriteStateTableTest.java

```diff
@@ -380,6 +385,77 @@ public void testCopyOnWriteContracts() {
 		Assert.assertTrue(originalState5 == stateTable.get(5, 1));
 	}
 
+	/**
```

Comment:
The `CopyOnWriteStateTableTest` could extend the `TestLogger`. That way we get better logging output on Travis.